### ISSUE TITLE
Backup comments to personal notes

### ISSUE THREAD

**Issue by timothyha** [ioparaskev/lichess_whisperer#1]
Lichess has a limit of 200 messages for the spectator chat box.  Maybe it would be good to at least back up ALL that the user has whispered into private notes of the game.  The long phrases would be preserved in full form, too.

Initially, I thought of console.log, but notes seem to be a better place.

**Comment by ioparaskev** [comment]
Yes that's a good idea. I'll look into it. From a quick look I couldn't find an easy way to update the notes or retrieve them but I'll dig more into this
